pytest/tests/mocknet/mirror_test.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/env python3
```

Comment:
Wouldn't it be clear to call this file `pytest/tests/mocknet/mirror.py` ?

Reply:
true, fixed